spec/requests/users/confirmations_spec.rb

```diff
@@ -4,8 +4,8 @@
 
 RSpec.describe "Users::Confirmations", type: :request do
   let(:confirmation_token) { "soC7rF8i_BgYdUxafFcP" }
-  let(:school) { create(:school) }
   let(:induction_coordinator) { user.induction_coordinator_profile }
+  let(:school) { induction_coordinator&.schools&.first }
```

Comment:
Do we need the null safety checks here?

Reply:
It's for `context "when user is not an induction coordinator" do`

Reply:
Not sure why this got updated. Are we reusing this in the new school journey? I expected it to be taken out in subsequent prs.

Reply:
It got updated because in one of the tests we test that an email was (or wasn't) sent to a particular school.

I think
```
expect(UserMailer).not_to receive(:primary_contact_notification).with(anything, anything)
```
is a lot broader than checking for an email from a particular school, so we shouldn't need `school` in that test.